community/modules/scripts/spack-setup/outputs.tf

```diff
@@ -49,3 +49,8 @@ output "spack_profile_script_path" {
   description = "Path to the Spack profile.d script."
   value       = var.spack_profile_script_path
 }
+
+output "system_user_name" {
+  description = "The system user used for Spack installation."
```

Comment:
I went with `The system user used to install Spack.`

I don't want to say "and Spack packages" because startup does not install packages (that would be the execute module).

I don't want to say "created" because this might be an account provided by the user that already existed.

Reply:
Also added "It can be reused by spack-execute module to install spack packages." to clarify.